### PR TITLE
Face detection using Viola-Jones algorithm in Matlab

### LINKED ISSUE #1752: face detection in an image using viola-jones algorithm
Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles to go over.

**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

*We're looking for the first 2-3 paragraphs of the article that appropriately summarize what your article will be about.*

**NOTE: We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

### Introduction paragraph (s)
Face detection is the ability to distinguish faces from non-faces object in an image or video. The viola-jones algorithm is an object recognition framework that allows the detection of the human faces. Despite being an outdated framework, viola-jones algorithm is quit power,robust and faster therefore it is found exceptionally notable in real-time face detection. In this article we will look on how to use viola-jones algorithm to detect faces in real time.

### Key takeaways: 
1. The reader shoould understand the need for face detection
2.  The reader should understand the viola-jones algorithm for face detection.
3. The reader should be able to use the algorithm to detect faces in an image

### References: 
Please list links to any published content/research that you intend to use to support/guide this article.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by jumapaul**
### Article checklist
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

This fixes #1752

### CODE REVIEW

File: articles/face-detection-matlab/index.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    label='no face detected';
+    imgn = insertText(img,position,label, 'fontsize',25,'BoxOpacity',1);
+    imshow(imgn)
+end
+```
+when we run the program, the following image is displayed on our figure window.
+
+![Faces detected](/engineering-education/face-detection-matlab/engineers.png)
+
+when we change the image and use a different image that has no face, the following is displayed on our figure window
```
Comment (kayere) [inline articles/face-detection-matlab/index.md:135]: Please add the image link.

**Review by kayere — COMMENTED** [review]
Hello @jumapaul - Great article, just make the requested change and the article will be good to go.

**Comment by jumapaul** [comment]
@kayere I have made the requested change.

**Review by kayere — APPROVED** [review]

**Comment by hectorkambow** [comment]
[Face detection using Viola-Jones algorithm in Matlab #1803.pdf](https://github.com/section-engineering-education/engineering-education/files/6225410/Face.detection.using.Viola-Jones.algorithm.in.Matlab.1803.pdf)


@jumapaul  

Upon running your article through our 3rd party plagiarism checker it seemed to raise a few flags and the % was higher than we typically accept. (we tend to strive for 10% or lower)

Please see attached PDF - and revisit the article to ensure we are contributing wholly unique and original content.

Be sure to see our resources page to see more info on plagiarism and what is considered as such. 🚀

**Comment by jumapaul** [comment]
@hectorkambow I have updated the article.

**Comment by hectorkambow** [comment]
plagiarism check done

**Pull request merged**